joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstForControlStructuresCreator.scala

```diff
@@ -112,10 +114,23 @@ trait AstForControlStructuresCreator(implicit withSchemaValidation: ValidationMo
   protected def astForTryStmt(stmt: PhpTryStmt): Ast = {
     val tryBody = stmtBodyBlockAst(stmt)
 
+    scope.pushNewScope(NewBlock())
     val catches = stmt.catches.map { catchStmt =>
       val catchNode = controlStructureNode(catchStmt, ControlStructureTypes.CATCH, "catch")
```

Comment:
Wondering if you could try build (and test) a more complete "code" property? Something like `s"catch (catchStmt.types.map(_.name).mkString(" | ") $catchStmt.variable)"`